src/content/ui/adaptive-responsive/large-screens.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Large screen devices
+description: >
+  Things to keep in mind when adapting apps to large screens.
+short-title: Large screens
+---
+
+<?code-excerpt path-base="ui/adaptive_app_demos"?>
+
+This page provides guidance on optimizing your
+app to improve its behavior on large screens.
+
+Flutter, like Android, defines [large screens][] as tablets,
+foldables, and ChromeOS devices running Android. Flutter
+_also_ defines large screen devices as web, desktop,
+and iPads.
+
+<b>PENDING: Mariam, do you have any specific iPad numbers?</b>
```

Comment:
according to statistica... in q4 of 2023 there were 14.9 million iPad users https://www.statista.com/statistics/299632/tablet-shipments-apple/

thats the best source i could find